jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcTriggerRepository.java

```diff
@@ -1,6 +1,7 @@
 package io.kestra.jdbc.repository;
 
 import io.kestra.core.models.QueryFilter;
+import io.kestra.core.models.QueryFilter.Field;   // ← ADDED
```

Comment:
```suggestion
import io.kestra.core.models.QueryFilter.Field;
```

Please avoid useless comments like that, there are other, please remove all of them